manifests/t/TeamDriveSystems/TeamDrive/4.8.0.3230/TeamDriveSystems.TeamDrive.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TeamDriveSystems.TeamDrive
+PackageVersion: 4.8.0.3230
+PackageLocale: en-US
+Publisher: TeamDrive Systems GmbH
+PublisherUrl: https://teamdrive.com
+# PublisherSupportUrl:
+PrivacyUrl: https://teamdrive.com/en/privacy-policy
+Author: TeamDrive Systems GmbH
+PackageName: TeamDrive Client
```

```suggestion
PackageName: TeamDrive
```
This must match Control Panel.

---

Thank you very much 👍